### DISCUSSION TITLE
Where to find list of icons?

### DISCUSSION THREAD

**Discussion by skorphil** [mantinedev/mantine#5206] [Q&A]
Hi, where can i find a full list of mantine icons? like `< IconAdjustments />` etc... look through documentation without success

**Comment by rtivital** [comment]
ActionIcon is not an icon, it is a button component – https://mantine.dev/core/action-icon/
For icons libraries see – https://mantine.dev/guides/icons/

**Reply by skorphil** [reply]
Yeah, i meant `<IconAdjustments />` Thank you
